src/components/forms/Currency/currency-utils.ts

```diff
@@ -22,33 +22,21 @@ export const getCurrencyOptions = (inISOCode: string) => {
     } else {
       // For all other ISO codes, use first 2 characters as the lookup from CurrencyMap
       const countryCode = inISOCode.substring(0,2);
+      // If the country code doesn't have an entry, default to US
       currMapToUse = CurrencyMapAlias[countryCode];
+      if (!currMapToUse) {
+        currMapToUse = CurrencyMapAlias['US'];
+      }
```

Comment:
I feel it's better to move code to line number 39 so that it will handle other unexpected cases.